cli/cli-client/src/main/java/org/infinispan/cli/completers/CdContextCompleter.java

```diff
@@ -51,7 +51,11 @@ public void complete(CompleterInvocation invocation) {
                if (parts[i].isEmpty()) {
                   resource = resource.findAncestor(RootResource.class);
                } else {
-                  resource = resource.getChild(parts[i]);
+                  try {
+                     resource = resource.getChild(parts[i]);
+                  } catch (IOException e) {
```

Comment:
Won't this just NPE in the next line instead though?

Reply:
resource won't be null because of L24

Reply:
I didn't look far enough up ;)